app/assets/stylesheets/partials/_response_modal.scss

```diff
@@ -0,0 +1,70 @@
+.response__modal {
+  background-color: $v3_color_weigh-in-response-bg;
+  border-radius: .5rem;
+  grid-template-columns: 10rem 1fr 10rem;
+  margin: 0 7rem;
+  padding: 2rem;
+
+  &-overlay {
+    background-color: rgba(63, 60, 60, 0.705);
+    height: 100%;
+    left: 0;
+    overflow: hidden;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 2;
```

Comment:
If you set this to `101` it should fix the header overlap problem.

Reply:
Done